src/org/traccar/processing/ComputedAttributesHandler.java

```diff
@@ -31,19 +31,34 @@
 import org.traccar.Context;
 import org.traccar.helper.Log;
 import org.traccar.model.Attribute;
+import org.traccar.model.Device;
 import org.traccar.model.Position;
 
 public class ComputedAttributesHandler extends BaseDataHandler {
 
     private JexlEngine engine;
 
+    private boolean mapDeviceAttributes;
+
     public ComputedAttributesHandler() {
         engine = new JexlEngine();
         engine.setStrict(true);
+        if (Context.getConfig() != null) {
+            mapDeviceAttributes = Context.getConfig().getBoolean(
+                    "processing.computedAttributes.mapDeviceAttributes.enable");
```

Comment:
Seems too long. Can we just name it `processing.computedAttributes.deviceAttributes`?